docs/schema/piechart/example.star

```diff
@@ -0,0 +1,17 @@
+load("render.star", "render")
```

Comment:
`make lint` will fix the formatting of this file, which is why the build is failing. It might be best to remove this example though - it's not schema and the widget examples get generated from the `EXAMPLE` you have below in `pie_chart.go`